.github/workflows/image_docker.yml

```diff
@@ -0,0 +1,25 @@
+name: Create projects test image
+
+on:
+  push:
+    paths:
+      - 'package.json'
+      - 'package-lock.json'
+      - 'Dockerfile'
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
```

Comment:
Conviene siempre que uséis la última versión disponible. ¿Has comprobado si esta lo es?

Reply:
:eyes: también estos.